holoviews/tests/operation/testdatashader.py

```diff
@@ -68,7 +78,7 @@ def test_aggregate_points_categorical(self):
     def test_aggregate_points_categorical_zero_range(self):
         points = Points([(0.2, 0.3, 'A'), (0.4, 0.7, 'B'), (0, 0.99, 'C')], vdims='z')
         img = aggregate(points, dynamic=False,  x_range=(0, 0), y_range=(0, 1),
-                        aggregator=ds.count_cat('z'))
+                        aggregator=ds.count_cat('z'), height=2)
```

Comment:
Why is there a new `height` argument in a test about aggregating points?

Reply:
Something was very wrong with the comparisons and I can't quite explain it. I updated comparisons to avoid flattening the array, so it is now comparing 2D arrays, which provides a much better error message. After doing that the test (correctly) started complaining that one array was (0, 400) and the other (0, 3), obviously when you flatten the arrays both are shape `(0,)` so it didn't error.

Reply:
Ah. Glad you caught that then!